telco-core/configuration/core-finish.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: core-customer-policies
+policyDefaults:
+  namespace: ztp-core-policies
+  policySets: []
+  placement:
+    clusterSelectors:
+      common: "core"
+      finish-version-4.18: ""
```

Comment:
Similar to core-overlay and core-baseline I think we want this to be `version: "4.18"`. If we want to bind only during initial cluster install we can add an exclusion for ztp-done.

Reply:
fixed. Note that core-upgrade has `upgrade-version-4-18: ""`